index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;700&display=swap" rel="stylesheet">
+    <title>Portfolio</title>
+
+    <link rel="stylesheet" href="./asset/css/style.css">
+</head>
+<body>
+    <header>
+        <div>
+            <span id="logo">My Logo</span>
+        </div>
+        <nav>
+            <img id="hamburger" src="./asset/images/menu.svg" alt="hamburger">
+        </nav>
+    </header>
+    <section id="main-section">
+        <div class="container">
+            <div id="profile-header">
+                <h1>I'm Muyiwa <br>Glad to see <br> you!</h1>
+            </div>
+            
+            <div id="profile-description">
+                <p>I'm a software developer! I can help you build a product , feature or website Look through some of my work and
+                experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+            </div>
+            <div id="connect">
+                <h3>Let's Connect</h3>
+            </div>
+            <div class="social-media-icons">
+                <a href="#"><img src="./asset/images/Vectortwitter.svg" alt="twitter"></a>
+                <a href="#"><img src="./asset/images/linkedin.svg" alt="linkedin"></a>
+                <a href="#"><img src="./asset/images/Vectorm-icon.svg" alt="M"></a>
+                <a href="#"><img src="./asset/images/Vectorgithub.svg" alt="github"></a>
+                <a href="#"><img src="./asset/images/Vectordribble.svg" alt="dribble"></a>
+            </div>
```

Comment:
This content should be wrapped in several `<li>` all of them wrapped in a `<ul>`. Please fix it.

<img width="603" alt="Screen Shot 2022-06-29 at 18 19 59" src="https://user-images.githubusercontent.com/82426682/176561696-d5661b70-f97e-41c3-9d7c-f4c46c0f4bb8.png">

Reply:
Done